### PR TITLE
Game Theory module rearrange + additions (still not a completed module though)

### PR DESCRIPTION
**Pull request by ssszlma**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by ssszlma** [comment]
Screenshots:

Old:
image

New:
image
image

**Comment by ssszlma** [comment]
(The screenshot cut off there's no duplicate of the cp-algo resource)

**Comment by envyaims** [comment]
why is easy ordered after normal

File: content/6_Advanced/Game_Theory.problems.json [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
   "MODULE_ID": "game-theory",
   "general": [
+    {
+      "uniqueId": "apple-tree-cf-gt",
+      "name": "Apple Tree",
+      "url": "https://codeforces.com/contest/812/problem/E",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": true,
+      "tags": [],
```
Comment (envyaims) [inline content/6_Advanced/Game_Theory.problems.json:11]: please put tags for your problems

**Review by envyaims — CHANGES_REQUESTED** [review]

**Comment by 4ierseries** [comment]
> why is easy ordered after normal

I forgot to change that 

I'll add the tags too thanks!

**Comment by envyaims** [comment]
> > why is easy ordered after normal
> 
> I forgot to change that :skull
> 
> I'll add the tags too thanks!

Can you be more specific with your tags? We know theyre game theory problems because theyre in the game theory module.. For example, first problem should have nim.

File: content/6_Advanced/Game_Theory.mdx [hunk 2]
```diff
@@ -2,35 +2,67 @@
 id: game-theory
 title: 'Game Theory'
 author: Benjamin Qi
-description: '?'
+contributors: Salma J
+description: 'Problems involving Algorithmic Game Theory'
```
Comment (envyaims) [inline content/6_Advanced/Game_Theory.mdx:6]:
```suggestion
description: 'Problems involving Algorithmic Game Theory'
```

well that's the name of the module.
I'd suggest something like "deducing optimal strategies" or something

File: content/6_Advanced/Game_Theory.mdx [hunk 3]
```diff
@@ -2,35 +2,67 @@
...
-  - [Factorization Game](https://www.hackerearth.com/problem/algorithm/mancunian-and-factorization-game-b8794702/)
-  - [Arpa & Game](http://codeforces.com/contest/850/problem/C)
-  - [Apple Tree](http://codeforces.com/contest/812/problem/E)
+<IncompleteSection />
+
+<Resources>
+	<Resource
+		source="HackerRank"
+		title="Game Theory and Grundy Numbers"
+		url="https://www.youtube.com/watch?v=JXTVOyQpSGM"
```
Comment (envyaims) [inline content/6_Advanced/Game_Theory.mdx:16]: wrong url

**Comment by ssszlma** [comment]
> Can you be more specific with your tags? We know theyre game theory problems because theyre in the game theory module.. For example, first problem should have nim, and others may have dp, etc.

Yeah I'll try doing that rn :)

File: content/6_Advanced/Game_Theory.mdx (hunk 3 above)
Comment (ssszlma) [reply content/6_Advanced/Game_Theory.mdx:16]: i didnt put that lmao it was just there, I'll try finding the og link

File: content/6_Advanced/Game_Theory.mdx (hunk 2 above)
Comment (ssszlma) [reply content/6_Advanced/Game_Theory.mdx:6]: Ok!

**Comment by SansPapyrus683** [comment]
nononono

**Comment by ssszlma** [comment]
Sorry I haven't checked this in a while 😭

**Review by SansPapyrus683 — APPROVED** [review]

**Review by envyaims — APPROVED** [review]

**Pull request merged**